mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -1650,6 +1676,7 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
                                     chatInput.focus();
                                 }
                             }}
+                            disabled={isSignedUp === false}
```

Comment:
These fields are not disabled or opaque either. One implementation idea this made me think of. Do you think it's possible to just disable the pointer events and add the opacity to the entire chat taskpane and all its children so we don't need to remember to do it everywhere? 

I think if we do something like, it would work? 

```css
.chat-taskpane *:not(.signup-form-container):not(.signup-form-container *) {
  color: blue;
  margin: 10px;
}
```

<img width="387" height="49" alt="Image" src="https://github.com/user-attachments/assets/0d1c3c56-e8bc-416c-a4d1-a8c8e773eafa" />